### PR TITLE
New Asset Section in Product Detail

### PR DESCRIPTION
**Pull request by asaadam**
inital commit for issue #133 

it's also refactoring component assets card.

for each item, now it's using `AssetsItem` Component.

### CODE REVIEW

**Comment by pyk** [comment]
@asaadam btw dam, currently we don't use `next/image` coz we export the site as static site

File: components/v1/MarketDetailPage/LeveragedTokenMyAssetCard.tsx [hunk 1]
```diff
@@ -35,32 +42,20 @@ const MyAssetCard: FunctionComponent<MyAssetCardProps> = ({ address }) => {
...
-                </div>
+            <div className="grid grid-cols-2 gap-8">
+                <AssetsItem title="Token Balance" image="/markets/tokenBalanceIcon.svg" value={`${tokenBalanceFormatter.format(balance)}`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Value (USDC)" image="/markets/valueIcon.svg" value={`${tokenBalanceFormatter.format(balance * nav)}`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Return" image="/markets/returnIcon.svg" value={`-`} showData={showData} showLoading={showLoading || showError} />
+                <AssetsItem title="Return (USDC)" image="/markets/returnDollarIcon.svg" value={`-`} showData={showData} showLoading={showLoading || showError} />
             </div>
+            <ButtonTertiary full onClick={async () => await addTokenToMetamask({ token: tokenType.ETHRISE, chainID: chain.chain.id })}>
+                Add {isVault ? metadata.vaultTitle : metadata.title}
+            </ButtonTertiary>
```
Comment (math-marcellino) [inline components/v1/MarketDetailPage/LeveragedTokenMyAssetCard.tsx]: 1. The `isVaultToken` arguments is missing when calling `addTokenToMetamask` function here, when we press `Add rvETHUSDC` button, it's still adding `ETHRISE` to Metamask.
2. I think the text inside the button should be `Add {$TOKEN} to Wallet`

Comment (asaadam) [reply components/v1/MarketDetailPage/LeveragedTokenMyAssetCard.tsx]: ahh ya I forgot to dynamically change token.

**Comment by asaadam** [comment]
> @asaadam btw dam, currently we don't use `next/image` coz we export the site as static site

copy ser

File: components/v1/MarketDetailPage/AssetsItem.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import { tokenBalanceFormatter } from "../../../utils/formatters";
```
Comment (pyk) [inline components/v1/MarketDetailPage/AssetsItem.tsx]: @asaadam It seems `tokenBalanceFormatter` is unused here

File: components/v1/MarketDetailPage/VaultInformationCard.tsx [hunk 3]
```diff
@@ -39,7 +40,7 @@ const VaultInformationCard: FunctionComponent<VaultInformationCardProps> = ({ ad
 
     return (
         <div className="flex w-full flex-col space-y-6 rounded-[16px] bg-gray-light-2 px-4 pb-4 dark:bg-gray-dark-2">
-            <div className="pt-4">
+            <div className=" pt-4">
```
Comment (pyk) [inline components/v1/MarketDetailPage/VaultInformationCard.tsx]: Remove this space

**Review by pyk — CHANGES_REQUESTED** [review]
@asaadam Just minor tweak and all good to go

**Review by pyk — APPROVED** [review]

**Pull request merged**